sig-services/service-requirements/security-implications.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Define and explain the security implications
+persona: service provider/owner
+phase: service-design, service-operational
+---
+
+The service should have defined security implications that are applied when a user subscribes to the service. The document that is describing these security implications should also define why are they required for the full functionality of the service.
```

Comment:
I would rephrase this, security implication isn't something that "is applied" I think.. it's rother something that "happens"...

```suggestion
The service should disclose all security implications that can have an impact on a user which subscribes to the service. These implications are listed in a document which describes in detail what is the risk and why it is necessary for the full functionality of the service. In case there's an option to opt-out of a particular risk factor the document states what effect it has on service functionality.
```

Reply:
Agree, the opt-out option is also a good idea.